### PR TITLE
Added WT69-A PCB

### PR DESCRIPTION
**Pull request by Wilba6582**
## Description

Added WT69-A PCB.

https://cdn.discordapp.com/attachments/393397694502010880/571594471112704020/i69_g.png

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/wilba_tech/wt69_a/config.h [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x6582 // wilba.tech
+#define PRODUCT_ID      0x069A // 69-A
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    wilba.tech
+#define PRODUCT         wilba.tech WT69-A
```
Comment (fauxpark) [inline keyboards/wilba_tech/wt69_a/config.h]:
```suggestion
#define PRODUCT         WT69-A
```

Comment (Wilba6582) [reply keyboards/wilba_tech/wt69_a/config.h]: `wilba.tech WT69-A` is how I want it to appear in the "Bluetooth & other devices" page in Windows 10, just like other devices combine manufacturer and product, i.e. `Logitech Unifying Receiver`,  `Acer XR341CK`, etc.

Comment (fauxpark) [reply keyboards/wilba_tech/wt69_a/config.h]: My G700s doesn't seem to do that (but it is combined in Device Manager), so it's not a surefire thing, but yeah, that will probably look much better.

Comment (Wilba6582) [reply keyboards/wilba_tech/wt69_a/config.h]: this is what I see (added "PRODUCT" to start of string):

https://puu.sh/Dkyu2/916b32356b.png

File: keyboards/wilba_tech/wt69_a/info.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "wilba.tech WT69-A", 
+    "url": "https://wilba.tech", 
+    "maintainer": "Wilba", 
+    "bootloader": "atmel-dfu", 
+    "width": 18.25, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt69_a/info.json]:
```suggestion
        "LAYOUT_all": {
```

File: keyboards/wilba_tech/wt69_a/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# WILBA.TECH WT69-A
+
+![WILBA.TECH WT69-A](https://wilba.tech)
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt69_a/readme.md]: URL here needs to be a direct image link, e,g.

```suggestion
![WILBA.TECH WT69-A](https://wilba.tech/images/wt69a.jpg)
```

File: keyboards/wilba_tech/wt69_a/readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+WT69-A is a keyboard PCB supporting 65% layout with 0.25U blocker and left side function keys. [More info at wilba.tech](https://wilba.tech/)
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt69_a/readme.md]: Render looks like a 0.5U blocker to me.

```suggestion
WT69-A is a keyboard PCB supporting 65% layout with 0.5U blocker and left side function keys. [More info at wilba.tech](https://wilba.tech/)
```

File: keyboards/wilba_tech/wt69_a/wt69_a.h [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, K015, K215, K016, \
+    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, K115,       K116, \
+    K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214,             K216, \
+    K300, K301, K302, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314,             K315, K316, \
+    K400, K401, K402, K403, K404,                   K408,                         K413,       K414, K415, K416  \
+) { \
+    { K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, K015, K016 }, \
+    { K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, K115, K116 }, \
+    { K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212, K213, K214, K215, K216 }, \
+	{ K300, K301, K302, ____, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314, K315, K316 }, \
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt69_a/wt69_a.h]: Tab &rarr; four Spaces, to match the other lines.

```suggestion
    { K300, K301, K302, ____, K304, K305, K306, K307, K308, K309, K310, K311, K312, K313, K314, K315, K316 }, \
```

File: keyboards/wilba_tech/wt65_a/readme.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
 # WILBA.TECH WT65-A
 
-![WILBA.TECH WT65-A](https://wilba.tech)
+![WILBA.TECH WT65-A](https://images.squarespace-cdn.com/content/5b267e429772ae4372e3b65f/1556352502542-AKHBF9R6CP5HVHOZYJHM/i69_g.png?content-type=image%2Fpng)
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt65_a/readme.md]: I think you meant to put this on `keyboards/wilba_tech/wt69_a/readme.md` Line 3.

Though this should be an image URL too.

```suggestion
![WILBA.TECH WT65-A](https://wilba.tech/images/wt65a.jpg)
```

File: keyboards/wilba_tech/wt69_a/readme.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# WILBA.TECH WT69-A
+
+![WILBA.TECH WT69-A](https://wilba.tech/images/wt69a.jpg)
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt69_a/readme.md]:
```suggestion
![WILBA.TECH WT65-A](https://images.squarespace-cdn.com/content/5b267e429772ae4372e3b65f/1556352502542-AKHBF9R6CP5HVHOZYJHM/i69_g.png?content-type=image%2Fpng)
```

File: keyboards/wilba_tech/wt65_a/readme.md (hunk 6 above)
Comment (Wilba6582) [reply keyboards/wilba_tech/wt65_a/readme.md]: whoops

**Comment by drashna** [comment]
Travis CI failure is unrelated to these changes. 

Looks good! Thanks!

**Pull request merged**